R/pivot-long.R

```diff
@@ -12,6 +12,10 @@
 #'   the cell values (`values_to`) turn into in the result.
 #'
 #'   Note that these variables do not exist, so must be specified as strings.
+#'
+#'   If `names_to` contains multiple columns (i.e., `c(".value", "time")`),
```

Comment:
It's only when you use the special `".value"` name that that you can omit the `values_to`. But I'm not sure how to phrase that.

Reply:
Ah- so for the case when you have multiple names to but you don't use the `.value`? The current wording sounds like you always need to specify both args as a pair, so it sounds like you won't need to use `values_to` if `names_to` contains multiple columns and one of those is the special variable name `.value`?

Reply:
Correct